code/src/components/HappyForm.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react"
+import "./happyForm.css"
+
+const url = "https://technigo-thoughts.herokuapp.com/"
+
+export const HappyForm = props => {
+  const [message, setMessage] = useState("")
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    fetch(url, {
+      method: "POST",
+      body: JSON.stringify({ message }),
+      headers: { "Content-Type": "application/json" }
+    })
+      .then(() => {
+        setMessage("")
+        props.onFormSubmit(message)
+      })
+      .catch(err => console.log("error:", err))
+  }
+
+  return (
+    <form className='happy-form'>
```

Comment:
Some places you are using `"` and some `'`. Please choose one and stick to it through out your project.